From 1c868d333f64df8c297e8d286a73bf5dee903a12 Mon Sep 17 00:00:00 2001
From: Miquel Raynal <miquel.raynal@bootlin.com>
Date: Thu, 18 Oct 2018 15:17:11 +0200
Subject: [PATCH 0717/1051] ARM64: dts: a3900-db: fix USB VBUS regulators gpios

USB VBUS regulator GPIOs are mpp44/45, not mpp12/13 (which are raw
NAND Ready/Busy lines, preventing at the same time this device to
work). Fix the DT entry.

Change-Id: Iabebdff81a3a150417d135ecfa218626524dc30c
Signed-off-by: Miquel Raynal <miquel.raynal@bootlin.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60386
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index e07062b0f98a..c56943eb9e87 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -29,7 +29,7 @@
 	};
 	cp0_reg_usb3_0_vbus: cp0_usb3_vbus@0 {
 		compatible = "regulator-fixed";
-		gpio = <&cp0_gpio1 12 GPIO_ACTIVE_HIGH>;
+		gpio = <&cp0_gpio2 12 GPIO_ACTIVE_HIGH>;
 		regulator-name = "cp0-xhci0-vbus";
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
@@ -38,7 +38,7 @@
 	};
 	cp0_reg_usb3_1_vbus: cp0_usb3_vbus@1 {
 		compatible = "regulator-fixed";
-		gpio = <&cp0_gpio1 13 GPIO_ACTIVE_HIGH>;
+		gpio = <&cp0_gpio2 13 GPIO_ACTIVE_HIGH>;
 		regulator-name = "cp0-xhci1-vbus";
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
-- 
2.17.1

